execution_workflows/QAPA/more_samplesheets/samplesheet_single.csv

```diff
@@ -1,7 +1,7 @@
 sample,fastq1,fastq2,bam,bai,gff,fasta,bed,mart_export
```

Comment:
Shouldn't the header only now be `sample,fastq1,fastq2`?

Reply:
I have changed this